### PR TITLE
Contact Section

### LINKED ISSUE #5: Create an Contact Me section with Social Links.
Create an Contact Me section with Social Links.

### PR DESCRIPTION
**Pull request by kaarthikrajeev**
## Things Accomplished
- create contact section .
- added links and some animation.
- contact form details save in google sheets.
- closes #5

### CODE REVIEW

**Comment by Adhithyarajan** [comment]
@kaarthikrajeev So beautiful, so elegent, just loOking like a WOW.

**Review by Adhithyarajan — COMMENTED** [review]
@kaarthikrajeev So beautiful, so elegant, just looking like a WOW.

**Review by Adhithyarajan — APPROVED** [review]

**Comment by kaarthikrajeev** [comment]
> @kaarthikrajeev So beautiful, so elegant, just looking like a WOW.

Thank you😇

File: public/index.html [hunk 1]
```diff
@@ -45,6 +45,38 @@ <h4 class="text-3xl p-2 font-medium">Software Developer</h4>
...
+                <a href="https://www.instagram.com/kar_92k/?igshid=OGQ5ZDc2ODk2ZA%3D%3D"><ion-icon class="h-12 text-blue-800 hover:text-red-400 rounded-2xl w-12 hover:bg-cyan-800 ease-in duration-500 hover:translate-y-2 hover:w-14 hover:h:14" name="logo-instagram"></ion-icon></a>
+                <a href="https://github.com/kaarthikrajeev"><ion-icon class="h-12 w-12 rounded-full text-blue-800 hover:text-red-400 hover:bg-cyan-800 ease-in duration-500 hover:translate-y-2 hover:w-14 hover:h:14" name="logo-github"></ion-icon></a>
+                <a href="https://www.linkedin.com/in/kaarthik-rajeev-charanthara-2367a3a1?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app"><ion-icon class="h-12 text-blue-800 w-12 hover:text-red-400 rounded-xl hover:bg-cyan-800 ease-in duration-500 hover:translate-y-2 hover:w-14 hover:h:14" name="logo-linkedin"></ion-icon></a>
+                <a href="mailto:karthick77r@gmail.com"><ion-icon class="h-12 text-blue-800 rounded-lg hover:text-red-400 w-12 hover:bg-cyan-800 ease-in duration-500 hover:translate-y-2 hover:w-14 hover:h:14" name="mail-unread-outline"></ion-icon></a>
```
[the remark this answers is not in the document]
Comment (ajayesivan) [reply public/index.html]: @kaarthikrajeev Please update all animation this way. You may have noticed that when hovering on the Download button it moves the elements in the right. Animating transition property will fix this issue. Along with that transition animations are generally more performant.

**Review by ajayesivan — APPROVED** [review]

**Pull request merged**